zstacklib/zstacklib/utils/ovn.py

```
@@ -464,6 +500,83 @@ def isOvsRunning(self):
+        packages = "dpdk openvswitch ovn ovn-host"
+        r, o, e = bash.bash_roe("yum --disablerepo=* --enablerepo=zstack-local "
+                                "--nogpgcheck install -y {}".format(packages))
+        if r != 0:
+            raise Exception('failed to install OVS/OVN packages: %s' % e)
+
+        # change ovs-vswitchd and ovn-controller user to root
+        r, o, e = bash.bash_roe("sed -i 's/^OVS_USER_ID=\"openvswitch:hugetlbfs\"/"
+                                "OVS_USER_ID=\"root:root\"/' /etc/sysconfig/openvswitch")
+        if r != 0:
+            raise Exception('failed to configure openvswitch service user: %s' % e)
+        r, o, e = bash.bash_roe("sed -i 's/^OVN_USER_ID=\"openvswitch:openvswitch\"/"
```

Ruff (0.15.12)

[warning] Unpacked variable `o` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)